py-polars/polars/dataframe/frame.py

```diff
@@ -2549,6 +2549,7 @@ def write_excel(
         hidden_columns: Sequence[str] | None = None,
         hide_gridlines: bool = False,
         sheet_zoom: int | None = None,
+        freeze_panes: str | tuple[int, int] | tuple[int, int, int, int] | None = None,
```

Comment:
Could also be `[str, int, int]` ;)

Reply:
Can it? It's not in their documentation, granted they don't really have a well-defined API: https://xlsxwriter.readthedocs.io/worksheet.html#freeze_panes. I suppose a simple test will tell. I have a meeting but I can do this in an hour.

Reply:
Yup; it's implicit in the first two parameter values being replaceable with "A1" notation - that remains true even if then add the optional final two.